src/rdata/rfc1035.rs

```diff
@@ -2093,14 +2152,19 @@ impl<'a> Iterator for TxtIter<'a> {
     type Item = &'a [u8];
 
     fn next(&mut self) -> Option<Self::Item> {
-        self.0.next().map(CharStr::into_octets)
+        self.0.next().map(CharStr::as_slice)
     }
 }
 
 //------------ TxtBuilder ---------------------------------------------------
 
+/// Iteratively build TXT record data.
+///
+/// This type allows building TXT record data by starting with a empty data
```

Comment:
I think this should say "starting with empty data" or "starting with an empty data".